code/datums/martial/tribal_claw.dm

```diff
@@ -0,0 +1,122 @@
+#define TAIL_SWEEP_COMBO "DDGH"
+#define FACE_SCRATCH_COMBO "HD"
+#define JUGULAR_CUT_COMBO "HHG"
+#define TAIL_GRAB_COMBO "DHGG"
+
+/datum/martial_art/tribal_claw
+    name = "Tribal Claw"
+    id = MARTIALART_TRIBALCLAW
+    allow_temp_override = FALSE
+    help_verb = /mob/living/carbon/human/proc/tribal_claw_help
+
+/datum/martial_art/tribal_claw/proc/check_streak(mob/living/carbon/human/A, mob/living/carbon/human/D)
+    if(findtext(streak,TAIL_SWEEP_COMBO))
+        streak = ""
+        tailSweep(A,D)
+        return TRUE
+    if(findtext(streak,FACE_SCRATCH_COMBO))
+        streak = ""
+        faceScratch(A,D)
+        return TRUE
+    if(findtext(streak,JUGULAR_CUT_COMBO))
+        streak = ""
+        jugularCut(A,D)
+        return TRUE
+    if(findtext(streak,TAIL_GRAB_COMBO))
+        streak = ""
+        tailGrab(A,D)
+        return TRUE
+    return FALSE
+
+/datum/martial_art/tribal_claw/proc/tailAnimate(mob/living/carbon/human/A)
+    set waitfor = FALSE
+    for(var/i in GLOB.cardinals)
+        if(!A)
+            break
+        A.setDir(i)
+
+//Tail Sweep, triggers an effect similar to Space Dragon's tail sweep but only affects stuff 1 tile next to you, basically 3x3.
+/datum/martial_art/tribal_claw/proc/tailSweep(mob/living/carbon/human/A, mob/living/carbon/human/D)
+    log_combat(A, D, "tail sweeped(Tribal Claw)")
+    D.visible_message("<span class='warning'>[A] sweeps [D]'s legs with their tail!</span>", \
+                        "<span class='userdanger'>[A] sweeps your legs with their tail!</span>")
+    tailAnimate(A)
+    var/obj/effect/proc_holder/spell/aoe_turf/repulse/spacedragon/R = new
+    var/list/turfs = list()
+    for(var/turf/T in range(1,A))
+        turfs.Add(T)
```

Comment:
orange would probably be better, since it doesn't include the turf you are standing on (Wouldn't this tail sweep you too.
Also would probably be easier just to called D.emote("spin")

Reply:
This doesn't sweep the user, space dragon sweep already makes you spin so i just removed all that code.